transifex/strings_en.json

```diff
@@ -1267,6 +1272,12 @@
         }
       }
     },
+    "DatasetDatasetSummary": {
```

Comment:
Would you mind moving src/components/dataset/dataset-summary.vue to src/components/dataset/summary.vue so that its name is correct in the Transifex JSON?